.env.baklavastaging

```diff
@@ -37,7 +37,7 @@ GETH_BOOTNODE_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/geth-all"
 GETH_BOOTNODE_DOCKER_IMAGE_TAG="a85db20179b503b802430aef7085d35aa761c63d"
 
 CELOTOOL_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/celo-monorepo"
-CELOTOOL_DOCKER_IMAGE_TAG="celotool-53016a800a510cddae73444377bcf366d753367a"
+CELOTOOL_DOCKER_IMAGE_TAG="celotool-99f83b1b5554ae6fb0233b7b0f8a236660cb34cd"
```

Comment:
TODO: This will need to be updated to a tag with a commit that exists in master and contains the new commands.